src/catalog/src/durable/transaction.rs

```diff
@@ -1533,7 +1534,61 @@ impl<'a> Transaction<'a> {
             Err(SqlCatalogError::UnknownNetworkPolicy(id.to_string()).into())
         }
     }
+    /// Removes the network policy `name` from the transaction.
+    ///
+    /// Returns an error if `name` is not found.
+    ///
+    /// Runtime is linear with respect to the total number of network policies in the catalog.
+    /// DO NOT call this function in a loop, use [`Self::remove_network_policies`] instead.
+    pub fn remove_network_policy(&mut self, name: &str) -> Result<(), CatalogError> {
```

Comment:
is this function used at all? note generally this function should be keyed on id, not name. I know `fn remove_role(...)` uses the name, but no idea why

Reply:
not used, just following the template. I'll remove.